server/README.md

```diff
@@ -164,6 +164,18 @@ To set the welcome theme, use the following environment value :
 To add a custom provider extend the Keycloak image and add the provider to the `/opt/jboss/keycloak/standalone/deployments/`
 directory.
 
+## Running custom scripts on startup
+
+To run custom scripts on container startup either extend the Keycloak image or use a volume, then add your script to
```

Comment:
As I mentioned on the mailing list (see http://lists.jboss.org/pipermail/keycloak-dev/2019-February/011642.html) we can't guarantee any stability of this scripts. We may change the location of the Keycloak distribution to some other directory for example and it is very likely that user script will break. We may also change the way the distribution itself is structured and that will also cause such scripts to fail.

I would explicitly like to make this statement part of the contract. In other words, please mention in the documentation that users who use this functionality do it without any guarantees.